war/src/main/less/modules/manage-jenkins.less

```diff
@@ -1,152 +1,175 @@
 /* ----------- Manage Jenkins page ----------- */
 
-.manage-messages {
-    margin-bottom: 20px;
-}
-
 .manage-messages a,
 .manage-messages a:visited {
-    color: inherit !important;
-    text-decoration: none;
+  color: inherit !important;
+  text-decoration: none;
 }
 
 .manage-messages a:hover,
 .manage-messages a:focus {
-    text-decoration: underline;
+  text-decoration: underline;
 }
 
 .manage-messages dl:first-child {
-    margin-top: 0;
+  margin-top: 0;
 }
 
 .manage-messages dl dt:first-child {
-    margin-top: 0;
+  margin-top: 0;
 }
 
 .manage-messages dl dt {
-    margin-top: 10px;
-    font-weight: normal;
+  margin-top: 10px;
+  font-weight: normal;
 }
 
 .manage-messages dl dd {
-    margin-left: 15px;
+  margin-left: 15px;
 }
 
 .manage-messages dl:last-child {
-    margin-bottom: 0;
+  margin-bottom: 0;
 }
 
 .manage-messages dl dt::after {
-    content: ": ";
+  content: ": ";
+}
+
+.manage-messages {
+  .alert:last-of-type {
+    margin-bottom: 30px;
+  }
 }
 
 .manage-messages .alert a {
-    text-decoration: underline;
+  text-decoration: underline;
 }
 
 .manage-messages .alert a:hover {
-    text-decoration: underline;
+  text-decoration: underline;
 }
 
 .manage-messages .alert form {
-    position: relative;
-    float: right;
-    margin: -6px 0 0 0 !important;
+  position: relative;
+  float: right;
+  margin: -6px 0 0 0 !important;
 }
 
 .manage-messages .alert form span {
-    margin: 0 0 0 4px !important;
+  margin: 0 0 0 4px !important;
 }
 
-.manage-option {
-    border: 1px solid transparent;
-    margin-bottom: 4px;
+.jenkins-settings {
```

Comment:
What does `.jenkins-settings` mean? Sorry for being cheeky, but if it's just for the manage page I'd just name it like it is, not a generic jenkins settings component.

Reply:
I would guess it would apply to the general settings page, or plugin manager for example.

Reply:
I named it `jenkins-settings` over `jenkins-manage/jenkins-manage-jenkins/manage-jenkins` or something along those lines as it's a touch shorter and more readable. I do recognise that it might not be instantly clear however, I think moving the file into a `pages` package over the current `modules` package could help devs understand that its styles for a specific page rather than a component?